site/content/docs/alerting/ssl-expiration.md

```diff
@@ -28,6 +28,12 @@ Simply create or pick an existing alert channel that your check subscribes to an
 ## API checks
 The domain for the certificate is parsed from the `URL` in the HTTP request settings so it does not require any setup.
 
+{{<info >}}
+**Error: unable to verify the first certificate.**
+If prompted with this error, the usual cause is the certificate chain of the given website being incomplete. This will not happen with a browser check, because the browser will complete it on its own. When running an API check, though, no browser is involved - therefore the error takes place.
```

Comment:
"...because the browser will complete it on its own." -> it is not clear what that "it" refers to. Let's change that to "certificate chain" to make this 100% unambiguous.